<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DOF_TeachClass</class>
 <widget class="QMainWindow" name="DOF_TeachClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DOF_Teach</string>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>891</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_6{
font: 12pt &quot;3ds&quot;;
}</string>
    </property>
    <property name="title">
     <string>Position Record</string>
    </property>
    <widget class="QTableWidget" name="PosView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>731</width>
       <height>341</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Node 1 POS</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Node 2 POS</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Node 3 POS</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID 1 POS</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <color alpha="200">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID 2 POS</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID 3 POS</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID 4 POS</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>22</y>
       <width>101</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_27">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnGetPos">
        <property name="styleSheet">
         <string notr="true">#btnGetPos{ 
/*border: none;*/
background-color:rgb(88, 88, 88);
color:white;
border-style:outset;
border-width:4px;
border-radius:4px;
border-color:rgba(255,255,255,30);
font:bold 12px;
/*color:rgba(0,0,0,100);*/
padding:6px;
} 
#btnGetPos:hover
{
background-color:rgb(56, 56, 56);
}</string>
        </property>
        <property name="text">
         <string>截取</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnDelete_Pos">
        <property name="styleSheet">
         <string notr="true">#btnDelete_Pos{ 
/*border: none;*/
background-color:rgb(88, 88, 88);
color:white;
border-style:outset;
border-width:4px;
border-radius:4px;
border-color:rgba(255,255,255,30);
font:bold 12px;
/*color:rgba(0,0,0,100);*/
padding:6px;
} 
#btnDelete_Pos:hover
{
background-color:rgb(56, 56, 56);
}</string>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnRecurring">
        <property name="styleSheet">
         <string notr="true">#btnRecurring{ 
/*border: none;*/
background-color:rgb(88, 88, 88);
color:white;
border-style:outset;
border-width:4px;
border-radius:4px;
border-color:rgba(255,255,255,30);
font:bold 12px;
/*color:rgba(0,0,0,100);*/
padding:6px;
} 
#btnRecurring:hover
{
background-color:rgb(56, 56, 56);
}</string>
        </property>
        <property name="text">
         <string>执行</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btnLoopRecurring">
        <property name="styleSheet">
         <string notr="true">#btnLoopRecurring{ 
/*border: none;*/
background-color:rgb(88, 88, 88);
color:white;
border-style:outset;
border-width:4px;
border-radius:4px;
border-color:rgba(255,255,255,30);
font:bold 12px;
/*color:rgba(0,0,0,100);*/
padding:6px;
} 
#btnLoopRecurring:hover
{
background-color:rgb(56, 56, 56);
}</string>
        </property>
        <property name="text">
         <string>LOOP</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="btnRecurring_Halt">
        <property name="styleSheet">
         <string notr="true">#btnRecurring_Halt{ 
/*border: none;*/
background-color:rgb(88, 88, 88);
color:white;
border-style:outset;
border-width:4px;
border-radius:4px;
border-color:rgba(255,255,255,30);
font:bold 12px;
/*color:rgba(0,0,0,100);*/
padding:6px;
} 
#btnRecurring_Halt:hover
{
background-color:rgb(56, 56, 56);
}</string>
        </property>
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="btnRecurring_UrgencyHalt">
        <property name="styleSheet">
         <string notr="true">#btnRecurring_UrgencyHalt{ 
/*border: none;*/
background-color:rgb(88, 88, 88);
color:white;
border-style:outset;
border-width:4px;
border-radius:4px;
border-color:rgba(255,255,255,30);
font:bold 12px;
/*color:rgba(0,0,0,100);*/
padding:6px;
} 
#btnRecurring_UrgencyHalt:hover
{
background-color:rgb(56, 56, 56);
}</string>
        </property>
        <property name="text">
         <string>急停</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>478</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCurrent">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#btnCurrent{ 
/*border: none;*/
background-color:rgb(26, 26, 26);
color:white;
border-style:outset;
border-width:4px;
border-radius:10px;
border-color:rgba(255,255,255,30);
font:bold 12px;
/*color:rgba(0,0,0,100);*/
padding:6px;
} 
#btnCurrent:hover
{
background-color:rgb(56, 56, 56);
}
/*QPushButton#fileBtn,#helpBtn,#settingBtn{  
    border: 1px solid #C0C0C0;  
    background-color:#C0C0C0;  
    border-style: solid;  
    border-radius:0px;  
    width: 40px;  //和你的控件大小一致
    height:20px;  //和你的控件大小一致
    padding:0 0px;  
    margin:0 0px;  
}  
QPushButton#expBtn{  
    border: 1px solid #C0C0C0;  
    background-color:#FFFFFF;  
    border-style: solid;  
    border-radius:0px;  
    width: 100px;  
    height:20px;  
    padding:0 0px;  
}  
QPushButton#fileBtn:hover,#helpBtn:hover,#settingBtn:hover,#expBtn:hover{     
    border: 1px solid #E3C46F;  
    background-color:#FEF4BF;  
    border-style: solid;  
    border-radius:2px;  
    width: 40px;  
    height:20px;  
    padding:0 0px;  
}  
QPushButton#fileBtn:pressed,#helpBtn:pressed,#settingBtn:pressed,#expBtn:pressed{  
    background-color:#EAF0FF;  
    border: 1px solid #AAB4C4;  
    width: 40px;  
    height:20px;  
    padding:0 0px;  
    border-radius:1px;  
}*/  </string>
       </property>
       <property name="text">
        <string>电流监控动态曲线</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnVelocity">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#btnVelocity{ 
/*border: none;*/
background-color:rgb(26, 26, 26);
color:white;
border-style:outset;
border-width:4px;
border-radius:10px;
border-color:rgba(255,255,255,30);
font:bold 12px;
/*color:rgba(0,0,0,100);*/
padding:6px;
} 
#btnVelocity:hover
{
background-color:rgb(56, 56, 56);
}</string>
       </property>
       <property name="text">
        <string>速度监控动态曲线</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnTorgue">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#btnTorgue{ 
/*border: none;*/
background-color:rgb(26, 26, 26);
color:white;
border-style:outset;
border-width:4px;
border-radius:10px;
border-color:rgba(255,255,255,30);
font:bold 12px;
/*color:rgba(0,0,0,100);*/
padding:6px;
} 
#btnTorgue:hover
{
background-color:rgb(56, 56, 56);
}
/*QPushButton#fileBtn,#helpBtn,#settingBtn{  
    border: 1px solid #C0C0C0;  
    background-color:#C0C0C0;  
    border-style: solid;  
    border-radius:0px;  
    width: 40px;  //和你的控件大小一致
    height:20px;  //和你的控件大小一致
    padding:0 0px;  
    margin:0 0px;  
}  
QPushButton#expBtn{  
    border: 1px solid #C0C0C0;  
    background-color:#FFFFFF;  
    border-style: solid;  
    border-radius:0px;  
    width: 100px;  
    height:20px;  
    padding:0 0px;  
}  
QPushButton#fileBtn:hover,#helpBtn:hover,#settingBtn:hover,#expBtn:hover{     
    border: 1px solid #E3C46F;  
    background-color:#FEF4BF;  
    border-style: solid;  
    border-radius:2px;  
    width: 40px;  
    height:20px;  
    padding:0 0px;  
}  
QPushButton#fileBtn:pressed,#helpBtn:pressed,#settingBtn:pressed,#expBtn:pressed{  
    background-color:#EAF0FF;  
    border: 1px solid #AAB4C4;  
    width: 40px;  
    height:20px;  
    padding:0 0px;  
    border-radius:1px;  
}*/  </string>
       </property>
       <property name="text">
        <string>转矩监控动态曲线</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>381</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox{
font: 12pt &quot;3ds&quot;;
}</string>
    </property>
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>351</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbActuate">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
        </property>
        <property name="text">
         <string>启动内驱力</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbRelate">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
        </property>
        <property name="text">
         <string>相对位移</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbTorgue">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
        </property>
        <property name="text">
         <string>启动力矩</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionConfig"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="DOF_Teach.qrc">
     <normaloff>:/icon/ico/exit_128px.ico</normaloff>:/icon/ico/exit_128px.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="DOF_Teach.qrc">
     <normaloff>:/icon/ico/usb_connected_128px_1168710_easyicon.net.ico</normaloff>:/icon/ico/usb_connected_128px_1168710_easyicon.net.ico</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>连接端口</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="icon">
    <iconset resource="DOF_Teach.qrc">
     <normaloff>:/icon/ico/data_128px_1156789_easyicon.net.ico</normaloff>:/icon/ico/data_128px_1156789_easyicon.net.ico</iconset>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
   <property name="toolTip">
    <string>关节参数配置</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="DOF_Teach.qrc">
     <normaloff>:/icon/ico/save_as_128px_1169862_easyicon.net.ico</normaloff>:/icon/ico/save_as_128px_1169862_easyicon.net.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>save data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="DOF_Teach.qrc">
     <normaloff>:/icon/ico/open_in_browser_128px_1169854_easyicon.net.ico</normaloff>:/icon/ico/open_in_browser_128px_1169854_easyicon.net.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>read data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DOF_Teach.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>DOF_TeachClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
